composer.json

```diff
@@ -15,11 +15,11 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "league/commonmark": "dev-master"
```

Comment:
The jump to `0.8` would be considered a major release which could break backwards compatibility, so `0.7.*` would be the safe constraint to use.  `~0.7` seems like a good compromise between that and `dev-master` though.